pkgs/servers/windmill/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, callPackage
+, pkg-config
+, openssl
+, python3
+, deno
+, librusty_v8 ? callPackage ../../development/web/deno/librusty_v8.nix { }
```

Comment:
Wth?

Reply:
This should be added to top-level or passthru.
Relying on the fallback is bad coding style.